features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -680,7 +680,7 @@
   "console.session.params.sessionRefreshTimeOut": 300,
   "console.session.params.checkSessionInterval": 3,
   "console.applications.enabled": true,
-  "console.product_version.configs.productVersion": "BETA",
+  "console.product_version.configs.productVersion": "",
```

Comment:
we don't need this from config in the default.json file.

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/4129/commits/ad2dd6a4a862977397856428e1c3f71d4646c784